projects/exercises/01/drive_in_line.cpp

```diff
@@ -0,0 +1,14 @@
+#include <STSL/RJRobot.h>
+
+using namespace std;
```

Comment:
I don't think we need this in this project.

Reply:
@chachmu can you remove this, then I'll merge it?